src/main/java/com/pnudev/librarysystem/util/SpecificationUtils.java

```diff
@@ -9,4 +9,10 @@
     public static <T> Specification<T> fieldContainsIgnoreCase(String field, String value){
```

Comment:
If you are going to fully "generify" these utilities, you might as well make them an abstract class and then extend it where needed
Also it should be named something like `AbstractSpecificationBuilder`

Reply:
Done, if I understood you correctly